diff --git a/node_modules/anilist-node/lib/media.js b/node_modules/anilist-node/lib/media.js
index bc9b502..0bf0345 100644
--- a/node_modules/anilist-node/lib/media.js
+++ b/node_modules/anilist-node/lib/media.js
@@ -38,7 +38,7 @@ class Media {
             characters { nodes { id name { english: full } } } staff { nodes { id name { english: full } } } studios { nodes { id name isAnimationStudio } } 
             isFavourite isAdult isLocked nextAiringEpisode { timeUntilAiring airingAt episode } airingSchedule { nodes { airingAt timeUntilAiring episode } }
             trends { nodes { date trending popularity inProgress } } externalLinks { url }
-            streamingEpisodes { title thumbnail url site } rankings { rank type context year season } mediaListEntry { id status }
+            streamingEpisodes { title thumbnail url site } rankings { rank type context year season } mediaListEntry { id status progress }
             reviews { nodes { id score summary body } } siteUrl autoCreateForumThread modNotes 
             stats { scoreDistribution { score amount } statusDistribution { status amount } }
             isRecommendationBlocked recommendations { nodes { mediaRecommendation { id title { romaji english native userPreferred } type } } } } }`,
