diff --git a/node_modules/anilist-node/lib/lists.js b/node_modules/anilist-node/lib/lists.js
index fdbcef0..50fed23 100644
--- a/node_modules/anilist-node/lib/lists.js
+++ b/node_modules/anilist-node/lib/lists.js
@@ -30,6 +30,7 @@ class Lists {
 				`lists { name isCustomList isSplitCompletedList status entries { id
             media { id idMal title { romaji english native userPreferred } 
             episodes description format startDate { year month day } endDate { year month day }
+						coverImage { large:extraLarge medium:large small:medium color }
             duration genres synonyms tags { name isMediaSpoiler } isFavourite isAdult siteUrl }
             status score progress repeat priority private notes hiddenFromStatusLists
             advancedScores startedAt { year month day } completedAt { year month day } updatedAt createdAt } } } }`,
diff --git a/node_modules/anilist-node/lib/media.js b/node_modules/anilist-node/lib/media.js
index bc9b502..0bf0345 100644
--- a/node_modules/anilist-node/lib/media.js
+++ b/node_modules/anilist-node/lib/media.js
@@ -38,7 +38,7 @@ class Media {
             characters { nodes { id name { english: full } } } staff { nodes { id name { english: full } } } studios { nodes { id name isAnimationStudio } } 
             isFavourite isAdult isLocked nextAiringEpisode { timeUntilAiring airingAt episode } airingSchedule { nodes { airingAt timeUntilAiring episode } }
             trends { nodes { date trending popularity inProgress } } externalLinks { url }
-            streamingEpisodes { title thumbnail url site } rankings { rank type context year season } mediaListEntry { id status }
+            streamingEpisodes { title thumbnail url site } rankings { rank type context year season } mediaListEntry { id status progress }
             reviews { nodes { id score summary body } } siteUrl autoCreateForumThread modNotes 
             stats { scoreDistribution { score amount } statusDistribution { status amount } }
             isRecommendationBlocked recommendations { nodes { mediaRecommendation { id title { romaji english native userPreferred } type } } } } }`,
